workflow-templates/im-test-cypress-schedule.yml

```diff
@@ -0,0 +1,203 @@
+          path: |
+            ${{ env.NPM_CACHE_PATH }}
+
+      - name: Authenticate with GitHub Packages
+        uses: im-open/authenticate-with-gh-package-registries@v1.1
+        with:
+          read-pkg-token: ${{ secrets.READ_PKG_TOKEN }} # This is an org-level secret
+          orgs: 'im-enrollment,im-practices' # TODO: Verify list of orgs packages will be pulled from
+
+      - name: NPM Install
+        if: steps.has-cache.outputs.cache-hit != 'true'
+        working-directory: '${{ env.CYPRESS_NPM_PACKAGE_PATH }}'
+        run: |
+          echo "Installing NPM"
+          npm ci
+
+      - name: Save cache for node_modules directory
+        if: steps.has-cache.outputs.cache-hit != 'true'
+        uses: actions/cache/save@v4
+        with:
+          key: ${{ env.NPM_CACHE_KEY }}
+          enableCrossOsArchive: true
+          path: |
+            ${{ env.NPM_CACHE_PATH }}
+
+  run-cypress:
+    runs-on: im-linux
+    needs: npm-cache
+    strategy:
+      matrix:
+        environment: [dev, qa, stage] # TODO: Add environments you want your tests to run in
+
+    defaults:
+      run:
+        shell: bash
+        working-directory: ${{ env.CYPRESS_TEST_DIR }}
+
+    environment: ${{ matrix.environment }}
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ env.GITHUB_REF }}
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Download npm cache
+        id: restore-npm-cache
+        uses: actions/cache/restore@v4
+        with:
+          key: ${{ needs.npm-cache.outputs.NPM_CACHE_KEY }}
+          enableCrossOsArchive: true
+          fail-on-cache-miss: true
+          path: |
+            ${{ env.NPM_CACHE_PATH }}
+
+      - name: Rebuild Node Modules
+        if: steps.restore-npm-cache.outputs.cache-hit == 'true'
+        working-directory: '${{ env.CYPRESS_NPM_PACKAGE_PATH }}'
+        run: npm rebuild
+
+      - name: Run cypress
+        id: cypress
+        continue-on-error: true
+        run: npm run ${{ env.CYPRESS_CI_SCRIPT }}
+        env:
+          APPLICATIONINSIGHTS_CONNECTION_STRING: ${{ secrets.APPLICATIONINSIGHTS_CONNECTION_STRING }}
+
+      - name: Merge json files and create html report
+        run: npm run ${{ env.CYPRESS_REPORT_SCRIPT }}
+
+      - name: Create Status check based on merged cypress results
+        id: process-cypress
+        uses: im-open/process-cypress-test-results@v3.0.2
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }} # Special per-job token generated by GH for interacting with the repo
+          results-file: '${{ env.CYPRESS_TEST_DIR }}/${{ env.CYPRESS_RESULTS_FILE }}'
+          report-name: 'Cypress ${{ github.run_number }}'
+          # TODO: Verify whether you want status checks or pr comments with the results.
+          create-status-check: false
+          create-pr-comment: false
```

Comment:
Since this is triggered by a scheduled job and not a pull request, I don't think anyone is going to want to set the values for these parameters to anything but `false`.

Reply:
I have removed the TODO comment from here also added comments for those two inputs `#This should be set as false as this is a scheduled job`